apps/app/src/pages/Build/index.js

```diff
@@ -376,24 +364,27 @@ const BuildContent = ({ ownerLogin, repositoryName, buildNumber }) => {
             mt={5}
             ref={observe}
           >
-            <Button
-              variant="neutral"
-              onClick={() => setShowStableScreenshots((prev) => !prev)}
-              justifyContent="start"
-            >
-              <IllustratedText
-                icon={showStableScreenshots ? EyeClosedIcon : EyeIcon}
-                field
+            {stats.stableScreenshotCount > 0 &&
+            stats.stableScreenshotCount < pageInfo.totalCount ? (
```

Comment:
I don't understand the second condition.

Reply:
If there is only stable screenshots we set the filter to `falseˋ and hide the toggle button

Reply:
ok